ElectronicObserver/Data/PoiDbSubmission/PoiDbRouteSubmission/Saku.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Text.Json.Serialization;
+
+namespace ElectronicObserver.Data.PoiDbSubmission.PoiDbRouteSubmission;
+
+public class Saku
```

Comment:
Maybe rename this class or add a comment to know what it is ?

Reply:
renamed